docs/examples/extensions/dashboard-section/js/global-prices.js

```diff
@@ -0,0 +1,43 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import moment from 'moment';
+
+/**
+ * WooCommerce dependencies
+ */
+import { Card, Chart } from '@woocommerce/components';
+import { formatCurrency } from '@woocommerce/currency';
+
+const data = [];
+
+for ( let i = 1; i <= 20; i++ ) {
+	const date = moment().subtract( i, 'days' );
+	data.push( {
+		date: date.format( 'YYYY-MM-DDT00:00:00' ),
+		primary: {
+			label: 'Global Apple Prices, last 20 days',
+			labelDate: date.format( 'YYYY-MM-DD 00:00:00' ),
+			value: Math.floor( Math.random() * 100 ),
+		},
+	} );
+}
+
+const GlobalPrices = () => {
+	return (
+		<Card className="woocommerce-dashboard__chart-block" title="Global Apple Prices">
```

Comment:
This chart looks different from the other ones:

![image](https://user-images.githubusercontent.com/3616980/58237782-41679180-7d46-11e9-8303-dcdac8d1a837.png)

It seems to be because it doesn't have the `woocommerce-analytics__card` class, was that in purpose?
```suggestion
		<Card className="woocommerce-dashboard__chart-block woocommerce-analytics__card" title="Global Apple Prices">
```

Reply:
Good one, looks way better now.